slave.mk

```diff
@@ -81,6 +81,8 @@ list :
 ## Include other rules
 ###############################################################################
 
+include $(RULES_PATH)/config
+
 ifeq ($(SONIC_ENABLE_PFCWD_ON_START),y)
```

Comment:
SONIC_ENABLE<feature_variables> are not taken effect from rules/config, as it is below these assignments.